docs/deployment/configuration.md

```diff
@@ -178,46 +178,49 @@ Here are the configuration keys, for both containers (environment variables) and
+| notification_manager:interval       | NOTIFICATION_MANAGER__INTERVAL                   | 10000                      | Interval to push notifications                      |
+| -                                   | -                                                | -                          | -                                                   |
+| publisher_manager:enabled           | PUBLISHER_MANAGER__ENABLED                       | `true`                     | Enable/disable the publisher manager                |
+| publisher_manager:lock_key          | PUBLISHER_MANAGER__LOCK_KEY                      | publisher_manager_lock     | Lock key for the manager in Redis                   |
+| publisher_manager:interval          | PUBLISHER_MANAGER__INTERVAL                      | 10000                      | Interval to send notifications / digests (in ms)    |
+| -                                   | -                                                | -                          | -                                                   |
+| ingestion_manager:enabled           | INGESTION_MANAGER__ENABLED                       | `true`                     | Enable/disable the ingestion manager                |
+| ingestion_manager:lock_key          | INGESTION_MANAGER__LOCK_KEY                      | ingestion_manager_lock     | Lock key for the manager in Redis                   |
+| ingestion_manager:interval          | INGESTION_MANAGER__INTERVAL                      | 300000                     | Interval to check for new data in remote feeds      |
+| -                                   | -                                                | -                          | -                                                   |
+| playbook_manager:enabled            | PLAYBOOK_MANAGER__ENABLED                        | `true`                     | Enable/disable the playbook manager                 |
+| playbook_manager:lock_key           | PLAYBOOK_MANAGER__LOCK_KEY                       | publisher_manager_lock     | Lock key for the manager in Redis                   |
+| playbook_manager:interval           | PLAYBOOK_MANAGER__INTERVAL                       | 60000                      | Interval to check new playbooks                     |
+| -                                   | -                                                | -                          | -                                                   |
+| file_index_manager:enabled          | FILE_INDEX_MANAGER__ENABLED                      | `true`                     | Enable/disable the file indexing manager            |
+| file_index_manager:lock_key         | FILE_INDEX_MANAGER__LOCK_KEY                     | file_index_manager_lock    | Lock key for the manager in Redis                   |
+| file_index_manager:interval         | FILE_INDEX_MANAGER__INTERVAL                     | 6000                       | Interval to check new files                         |
```

Comment:
There is a configuration missing : 
`"stream_lock_key": "file_index_manager_stream_lock"` => file_index_manager:stream_lock_key

Default interval is going to change, shouldn't be 6000 (@richard-julien)

Reply:
Put to 60000 as we said!